### DISCUSSION TITLE
How to master GitHub

### DISCUSSION THREAD

**Discussion by denisesarona** [community/community#75144] [New to GitHub]
How can i utilize GitHub for my studies?

**Comment by ktauchathuranga** [comment]
**practice?**

**Comment by Akash1134** [comment]
Welcome to GitHub and the GitHub Community @denisesarona  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by lamrelia** [comment]
Also check out [The Odin Project Git Basics](https://www.theodinproject.com/paths/foundations/courses/foundations#git-basics), if you prefer a guided curriculum.
